site-src/contributing/devguide.md

```diff
@@ -54,10 +52,18 @@ git clone https://github.com/kubernetes-sigs/gateway-api
 cd gateway-api
 ```
 
-This project works with Go modules; you can chose to setup your environment
+This project works with Go modules; you can choose to setup your environment
 outside $GOPATH as well.
 
-### Building the code
+
+### Build the code
+
+To build the code, it needs `shasum`. On Fedora and RHEL, this command is called sha1sum. 
```

Comment:
It's not a huge deal, but this isn't accurate: the `sha1sum` command on Linux systems like Fedora and RHEL is part of the `coreutils` and is a binary for computing and checking SHA1 message digests. The `shasum` command on the other hand is part of the `perl-digest-SHA` package, and is not a binary but a [Perl](https://www.perl.org/) script which can handle multiple kinds of digests.

However you are right that it's kinda a "surprise" dependency. I would recommend you document the need for it and the explicit ways in which it can be installed at least for the systems you are familiar with: others can come along and document other systems they know later.

Reply:
@shaneutt Thanks for the review. Yes, that's right. I will change this shortly.